platformsh/Dockerfile

```diff
@@ -8,7 +8,9 @@ ENV PATH="/usr/local/bin/:${PATH}"
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-RUN apk add --update -q --no-cache bash build-base curl git libffi-dev make openldap-dev openssh-client php7 && \
+RUN apk add --update -q --no-cache bash build-base curl git libffi-dev make openldap-dev openssh-client python3-dev py3-pip && \
```

Comment:
Maybe we should just add python 3 (if really required) and platformsh cli to the php image?